src/theme/ThemeProvider.js

```diff
@@ -6,7 +6,8 @@ const theme = {
     background: '#2D2937',
     secondaryBackground: '#1D1B24',
     foreground: '#4F4A59',
-    warning: '#E03131',
+    warning: '#EF6C6C',
+    secondaryWarning: '#F18383',
```

Comment:
I don't really think we need two warning colours @jvssicawu can we just use one?

Reply:
yee one is fine for livesite, Im not sure if we use secondary warning for application form though

Reply:
Seems like the hacker app uses `#F18383` though - should we keep it, or change it to `#EF6C6C` to stay consistent? They are quite close.
<img width="238" alt="Screen Shot 2020-11-15 at 12 04 56 AM" src="https://user-images.githubusercontent.com/38872354/99179859-33de5e00-26d6-11eb-99ef-4c64ca41036a.png">

Reply:
i did it because of the _slightly_ different shades here, so i thought we might want two warning colours in general in the future! i'm happy to make the change (to one warning colour), @jvssicawu can you please re-confirm if that's ok? ty!!! 
![image](https://user-images.githubusercontent.com/47487758/99215172-09001280-2787-11eb-8e1a-0610324b646e.png)

Reply:
They look identical to me heH

Reply:
HAHA same (that's why i added it so late, i realized like last min) 
They're very similar but i checked the figma css and they be different 😢  Will wait on design sign-off but ya i agree i think 1 warning colour is fine

Reply:
I'm not sure why there's two, I can check with Tiff but since the difference is so minute i dont think it was intentional. you can change to use just one for now imo

Reply:
oh actually LMAO sorry i took a second look because my brain kinda slow at 1am but seems like its specifically for the hover/focus effect so yea its probably intentional actually